src/Rezoom.SQL.Compiler/ModelChange.fs

```diff
@@ -64,6 +69,8 @@ type private ModelChange(model : Model, inference : ITypeInferenceContext) =
                 return! ModelOps.dropView objName
             | DropTable ->
                 return! ModelOps.dropTable objName
+            | DropSchema ->
+                return! ModelOps.dropSchema { Value = Some drop.ObjectName.ObjectName; Source = objName.Source }
```

Comment:
@rspeele: Note that it is `ObjectName` rather than `SchemaName` (used for schema qualified objects).